src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,175 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { bodySmall, buttonMedium, header1 } from 'nft/css/common.css'
+import { vars } from 'nft/css/sprinkles.css'
+import { fetchTrendingCollections } from 'nft/queries'
+import { TimePeriod, TrendingCollection } from 'nft/types'
+import { formatEthPrice } from 'nft/utils/currency'
+import { putCommas } from 'nft/utils/putCommas'
+import { formatChange, toSignificant } from 'nft/utils/toSignificant'
+import { useEffect, useState } from 'react'
+import { useQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+
+import * as styles from './Banner.css'
+
+const Banner = () => {
+  /* Sets initially displayed collection to random number between 0 and 4  */
+  const [current, setCurrent] = useState(Math.floor(Math.random() * 5))
+  const [hovered, setHover] = useState(false)
+  const { data: collections } = useQuery(
+    ['trendingCollections'],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+    }
+  )
+
+  useEffect(() => {
+    /* Rotate through Top 5 Collections on 15 second interval */
+    let stale = false
+    if (hovered || stale) return
+    const interval = setInterval(async () => {
+      if (collections) {
+        const nextCollectionIndex = (current + 1) % collections.length
+        setCurrent(nextCollectionIndex)
+      }
+    }, 15_000)
+    return () => {
+      stale = true
+      clearInterval(interval)
+    }
+  }, [current, collections, hovered])
+
+  return (
+    <Box onMouseEnter={() => setHover(true)} onMouseLeave={() => setHover(false)} cursor="pointer" width="full">
+      {collections && collections[current] ? (
+        <Link to={`/nfts/collection/${collections[current].address}`} style={{ textDecoration: 'none' }}>
+          <Box style={{ height: '386px' }}>
```

Comment:
- can you avoid inline styles here?
- is this height as spec'd? it's not divisible by 4

Reply:
This was built against a previous Figma version (before common design system values were applied), and will be updated in a future PR along with a bunch of other layout/style updates. This PR is more of a first step towards migration than the final result.